### PR TITLE
Impact changes

### PR DESCRIPTION
**Pull request by danielskatz**
weaves impact into plan, in chapter 3 where impact figure is, in chapters 4-7, after each activity, and in chapter 10 (partly) as overall table

### CODE REVIEW

File: 03-plan.Rmd [hunk 1]
```diff
@@ -203,44 +203,89 @@ create and maintain that software.
 
 For software, URSSI aims to improve the sustainability of research software by
 
-* Developing and sharing good-enough and best practices for software projects,
-including for testing, governance, codes of conduct, continuous integration
+* Developing and curating best practices for software projects,
+including examing practices in industry,
```
Comment (karthik) [inline 03-plan.Rmd]: examining

File: 03-plan.Rmd [hunk 2]
```diff
@@ -203,44 +203,89 @@ create and maintain that software.
...
 For software, URSSI aims to improve the sustainability of research software by
 
-* Developing and sharing good-enough and best practices for software projects,
-including for testing, governance, codes of conduct, continuous integration
+* Developing and curating best practices for software projects,
+including examining practices in industry,
+for testing, governance, codes of conduct, continuous integration.
 
-* Documenting sustainability models and providing guidance on how to choose among them
+* Sharing information and pratices through communities calls, training
```
Comment (karthik) [inline 03-plan.Rmd]:
```suggestion
* Sharing information and practices through communities calls, training
```

File: 03-plan.Rmd [hunk 3]
```diff
@@ -203,44 +203,89 @@ create and maintain that software.
...
-* Promoting the publication of research software
+* A summer school, and the incubator program
 
-* Coordinating community efforts, including by URSSI, to create and provide training
-for those who want to develop and maintain research software
+* Promoting new career paths for those who develop and maintain research software
+
+* Developing and advocating for research software usage and impact metrics to be
+a factor in the hiring and promotion of software developers and maintainers,
+includind promoting the publication of research software
```
Comment (karthik) [inline 03-plan.Rmd]:
```suggestion
including promoting the publication of research software
```

File: 10-metrics-eval.Rmd [hunk 4]
```diff
@@ -1,3 +1,34 @@
...
 This chapter is being developed.
+
+This table maps URSSI activities to the three portions of URSSI's intended impact. Activities are
+preceded by a letter to indicate what area of URSSI they are from
+(C = community and outreach,
+E = education and training,
+I = incubator,
+P = policy), 
+and in the impact cells, X indicates a designed primary impact on an activity, and y indicates
+a designed secondard impact.
```
Comment (karthik) [inline 10-metrics-eval.Rmd]:
```suggestion
a designed secondary impact.
```

File: 07-Policy.Rmd [hunk 5]
```diff
@@ -131,23 +131,28 @@ Initially, the policy area will work to address the following set of aims:
 1. Establish career paths (including titles and evaluation criteria for hiring and promotion).
 
   - addresses Changes 2, 5 (from list of changes at the start of this chapter)
+  - aims primrily at the people and careers portion of URSSI's impact goal, and secondarily at the research software ecosystem portion
 
 2. Improve the measurement of impact of individuals, especially in activities that are inherently
 collaborative like software development. 
 
   - addresses Changes 1, 2, 3
+  - aims primrily at the people and careers portion of URSSI's impact goal, and secondarily at the research software ecosystem portion
```
Comment (karthik) [inline 07-Policy.Rmd]:
```suggestion
  - aims primarily at the people and careers portion of URSSI's impact goal, and secondarily at the research software ecosystem portion
```

File: 07-Policy.Rmd [hunk 6]
```diff
@@ -131,23 +131,28 @@ Initially, the policy area will work to address the following set of aims:
 1. Establish career paths (including titles and evaluation criteria for hiring and promotion).
 
   - addresses Changes 2, 5 (from list of changes at the start of this chapter)
+  - aims primrily at the people and careers portion of URSSI's impact goal, and secondarily at the research software ecosystem portion
```
Comment (karthik) [inline 07-Policy.Rmd]:
```suggestion
  - aims primarily at the people and careers portion of URSSI's impact goal, and secondarily at the research software ecosystem portion
```

**Comment by JeffCarver** [comment]
Changes to the Education chapter (5) look good to me. Thanks.

**Review by sandragesing — COMMENTED** [review]
The collaborations workshop is a good workshop but I think we might want to choose a workshop/conference organized by one of the other NSF institutes to foster the collaboration between the NSF institutes. The fellows could jump in for projects that cannot be filled by fellows of other institutes because it might be a more generic project.

**Review by sandragesing — COMMENTED** [review]
The four major outreach channels have only Slack as interactive channel. I know that there was not much uptake on discourse. It might be different if the first summerschool and carpentries have taken place and people start to discuss more about certain topics and don't want to lose the discussion. Maybe we could mention that we consider to offer it if we see the demand?

**Comment by danielskatz** [comment]
> The collaborations workshop is a good workshop but I think we might want to choose a workshop/conference organized by one of the other NSF institutes to foster the collaboration between the NSF institutes. The fellows could jump in for projects that cannot be filled by fellows of other institutes because it might be a more generic project.

The collaborations workshop is not discussed in this PR.  It's in existing (and unchanged) text in the Community and Outreach Chapter, and I think it's there as an example, not as something we would do. My request is that you open a new issue for this, and assign it to @karthik, as that's in his chapter, not in this PR.

**Comment by danielskatz** [comment]
> The four major outreach channels have only Slack as interactive channel. I know that there was not much uptake on discourse. It might be different if the first summerschool and carpentries have taken place and people start to discuss more about certain topics and don't want to lose the discussion. Maybe we could mention that we consider to offer it if we see the demand?

As far as I can tell, this is also not discussed in this PR. It's in existing (and unchanged) text in the Community and Outreach Chapter. My request is that you open a new issue for this, and assign it to @karthik, as that's in his chapter, not in this PR.

**Pull request merged**